src/svg-quirks-mode/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
Agreed. We won't be able to mix APL with BSD-3-Clause. We may want to bring in the fonts as a dependency rather than checking them in.